selfdrive/car/fw_versions.py

```diff
@@ -47,7 +45,13 @@ def build_fw_dict(fw_versions: List[capnp.lib.capnp._DynamicStructBuilder],
   return dict(fw_versions_dict)
 
 
-def match_fw_to_car_fuzzy(live_fw_versions, match_brand=None, log=True, exclude=None):
+class MatchFwToCar(Protocol):
```

Comment:
I had to use this instead of the Callable type because they are kwargs
https://stackoverflow.com/a/57840786/1908134